chaosinstana/probes.py

```diff
@@ -52,3 +108,27 @@ def convert_time(source_time: str) -> int:
             logger.error("Invalid date string provided")
             raise ActivityFailed("Invalid date string provided")
         return int(start_dt.timestamp() * 1000)
+
+
+def has_severity(event_data, severity_level):
+    """
+    Loops thorugh event data (expecting json array) and
+    looks for a severity element, if that element equals the severity_level
+    parameter given it returns True, if it reaches the end of the loop without
+    finding the severity_level it returns False. Any expection also return
+    false
+    """
+    try:
+        for event in event_data:
+            severity = event.get("severity")
+            if severity == severity_level:
+                logger.debug("has_severity for severity_level: {} is True"
+                             .format(severity_level))
+                return True
+
+        logger.debug("has_severity for severity_level: {} is False"
+                     .format(severity_level))
+        return False
+    except ValueError:
+        logger.error("has_severity error parsing event_data")
```

Comment:
shouldn't this be raising an ActivityFailed? I'm not sure.

Reply:
but the log should be a debug one considering the user cannot action it

Reply:
I have changes the logging to debug, I am not sure if this should raise an ActivityFailed, the exception would get raised if when parsing the instana response looking for severity, I have treated it as a if the severity level  was not found - this does not seem correct but not sure how to correct it

Reply:
amended logging to debug logging

Reply:
after discussion with russmiles we decided this was not an ActivityFailed, and just a debug log entry